library/src/main/java/com/firebase/ui/FirebaseRecyclerViewAdapter.java

```diff
@@ -133,7 +133,7 @@ public void onChanged(EventType type, int index, int oldIndex) {
      *                   combination of <code>limit()</code>, <code>startAt()</code>, and <code>endAt()</code>
      */
     public FirebaseRecyclerViewAdapter(Class<T> modelClass, int modelLayout, Class<VH> viewHolderClass, Firebase ref) {
```

Comment:
Let's bite the bullet and change the name to `FirebaseRecyclerAdapter` to a) be shorter and b) be in line with FirebaseListAdapter.